books/free-programming-books-id.md

```diff
@@ -87,6 +88,11 @@
 * [Tutorial Dasar Javascript untuk Pemula](https://www.petanikode.com/tutorial/javascript/) *(Dalam Proses)*
 
 
+### Laravel
+
+* [Lumen Fundamental](https://santrikoding.com) - Yudi Purwanto(HTML) [(PDF)](https://santrikoding.com/account/ebook/lumen-fundamental) _(Membutuhkan akun santrikoding atau email yang valid)_
```

Comment:
A bit of help according previous review

```suggestion
* [Lumen Fundamental](https://santrikoding.com/ebook/lumen-fundamental) - Yudi Purwanto (PDF) *(Membutuhkan akun santrikoding atau email yang valid)*
```

Reply:
I don't think we can accept a link that points to a login page with no information about the book we want to link to.